.travis.yml

```diff
@@ -19,7 +19,7 @@ node_js: 12
 
 _ios: &_ios
```

Comment:
can you remove the `sudo: false` key in line 1? Travis says it's a deprecated key. Also, the file is missing the license header.

Reply:
That should be separate PR as it had existed and I am not changing that specific area.